ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DirectionOrderMapper.java

```diff
@@ -46,6 +46,14 @@ public DirectionOrder from(CCDDirectionOrder ccdDirectionOrder, CCDOrderGenerati
         Optional.ofNullable(directionOrderData.getPaperDetermination()).ifPresent(paperDetermination ->
             builder.paperDetermination(YesNoOption.valueOf(paperDetermination.name())));
 
+        Optional.ofNullable(directionOrderData.getExpertReportPermissionPartyGivenToClaimant())
+            .ifPresent(permission -> builder
+                .expertReportPermissionGivenToClaimant(YesNoOption.valueOf(permission.name())));
+
+        Optional.ofNullable(directionOrderData.getExpertReportPermissionPartyGivenToDefendant())
+            .ifPresent(permission -> builder
+                .expertReportPermissionGivenToDefendant(YesNoOption.valueOf(permission.name())));
```

Comment:
Please add these in assertion and unit test.

Reply:
added